frontend/package.json

```diff
@@ -13,16 +13,13 @@
     "classnames": "2.2.6",
     "crypto-js": "4.0.0",
     "dequal": "1.0.0",
-    "epsg": "0.5.0",
```

Comment:
Did you use a tool to determine which are not used?  Or just a global search?

Reply:
depcheck